docs/pom.xml

```diff
@@ -61,5 +61,6 @@
         <module>troubleshooting-guide</module>
         <module>upgrade-guide</module>
         <module>distribution</module>
+        <module>EclipseGlassFishHelloSample</module>
```

Comment:
Please rename the module so that it's similar to names of other modules. For example, this would fit more: `eclipse-glassfish-hello-sample-app`, or simply `sample-app`

Reply:
I took sample-app